### DISCUSSION TITLE
找不到.ddns_go_config.yaml配置文件

### DISCUSSION THREAD

**Discussion by ucyo123** [jeessy2/ddns-go#1109] [Q&A] [answered]
本地pve下debian12安装的，因为卸载服务后又重新安装了一次，上一次设置了用户名和密码。
这次登录时候需要用户名和密码，忘记了用户名和密码。
在/root/下没有.ddns_go_config.yaml配置文件，请问下配置文件在哪里呢

**Comment by jeessy2** [comment, marked as answer]
没自定义配置路径的话，默认当前用户目录下，是一个隐藏文件。ls /root -al

**Reply by ucyo123** [reply]
谢谢，已经找到了，也看到了用户名和密码。
